items/FRACTURED_MITHRIL_PICKAXE.json

```diff
@@ -1,7 +1,7 @@
 {
   "itemid": "minecraft:stone_pickaxe",
-  "displayname": "§f§f§aFractured Mithril Pickaxe",
```

Comment:
Whats the change here?

Reply:
Just removed the §f§f because Hypixel apparently did

Reply:
> Just removed the §f§f because Hypixel apparently did

That just depends if you copy it from the AH or in your Hand.